bazel/repository_locations.bzl

```diff
@@ -25,7 +25,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "bazel-toolchains",
         project_desc = "Bazel toolchain configs for RBE",
         project_url = "https://github.com/bazelbuild/bazel-toolchains",
-        version = "3.6.0",
+        version = "3.7.1",
+        # TBD, not here yet
```

Comment:
Raised an inquiry at https://github.com/bazelbuild/bazel/issues/12188#issuecomment-733802361 since this hasn't shown up yet, the day after the 3.7.1 bazel release.

Reply:
Version 3.7.0 which exists already should be usable with Bazel 3.7.x

Reply:
Versions 3.7.0 and 3.7.1 both had build time regressions in envoy.

Version 3.7.0 and earlier entirely failed to invoke bazel on Windows when bazel would discover llvm 11.0.0 (earlier llvm 10 flavors were not an issue.)

The bazel 3.7.2 release solves both of these issues, based on initial review of rc1.